crates/rooch-types/src/bitcoin/genesis.rs

```diff
@@ -24,12 +25,16 @@ impl MoveStructState for BitcoinGenesisContext {
     fn struct_layout() -> move_core_types::value::MoveStructLayout {
         move_core_types::value::MoveStructLayout::new(vec![
             move_core_types::value::MoveTypeLayout::U8,
+            move_core_types::value::MoveTypeLayout::U64,
         ])
     }
 }
 
 impl BitcoinGenesisContext {
-    pub fn new(network: u8) -> Self {
-        Self { network }
+    pub fn new(network: u8, genesis_block_height: u64) -> Self {
+        Self {
+            network,
```

Comment:
After removing the cli and passing in bitcoin_network, is there a one-to-one correspondence between bitcoin_network and RoochChainID?

Reply:
Yes. We can use a custom chain to bind any bitcoin network to any rooch chain config.